client/go/dogma/cmd/watch_cmd.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/urfave/cli"
+	"sync"
+	"strings"
+)
+
+type watchCommand struct {
+	repo      repositoryRequestInfo
+	jsonPaths []string
+}
+
+func (wc *watchCommand) execute(c *cli.Context) error {
+	repo := wc.repo
+	client, err := newDogmaClient(c, repo.remoteURL)
+	if err != nil {
+		return err
+	}
+
+	query := createQuery(repo.path, wc.jsonPaths)
+	fw, err := client.FileWatcher(repo.projName, repo.repoName, query)
```

Comment:
I think we should not use `FileWatcher` here but just the low-level watch API, so that a user can specify `--last-known-rev` from CLI.

Reply:
Because I added a `streaming` option, I left the `FileWatcher` for less codes. But I can change using the low-level watch API when the `streaming` option is not specified.

Reply:
SGTM.